engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/meta/GetTypeConstructorsNode.java

```diff
@@ -0,0 +1,31 @@
+package org.enso.interpreter.node.expression.builtin.meta;
+
+import com.oracle.truffle.api.dsl.Fallback;
+import com.oracle.truffle.api.dsl.Specialization;
+import com.oracle.truffle.api.nodes.Node;
+import org.enso.interpreter.dsl.BuiltinMethod;
+import org.enso.interpreter.runtime.data.Array;
+import org.enso.interpreter.runtime.data.Type;
+
+@BuiltinMethod(
+    type = "Meta",
+    name = "get_type_constructors",
+    description = "Gets the constructors of a type.",
+    autoRegister = false)
+public abstract class GetTypeConstructorsNode extends Node {
+  static GetTypeConstructorsNode build() {
+    return GetTypeConstructorsNodeGen.create();
+  }
+
+  abstract Array execute(Object type);
+
+  @Specialization
+  Array doStruct(Type type) {
+    return new Array(type.getConstructors().values().toArray());
+  }
+
+  @Fallback
+  Array empty(Object type) {
+    return Array.empty();
```

Comment:
IMO this should throw a Type Error instead implicitly returning a fallback value. Is calling `get_type_constructors` on a non-Type ever a valid thing to do?

Reply:
`get_type_constructors` is a private function and nobody is supposed to call it. The call from `Meta.Type.constructors` shall be fine under the assumption the `Meta.Type.Value` was constructed properly with a "type".

Reply:
Done in 9033bc32c61e